js/cash_register.js

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
Wrap all your code in a function to create a closure so that we keep all these functions and variables private.